### ISSUE TITLE
Attention scores

### ISSUE THREAD

**Issue by jeroenmollink** [mahmoodlab/CLAM#2]
Thanks for sharing this great repository, it works really well! I was wondering if you could provide some guidelines how to extract the attention scores from the bag features for a single image?

Jeroen

**Comment by fedshyvana** [comment]
Hi Jeroen, you can just pass it to the model and the attention scores is returned along with other outputs:
logits, Y_prob, Y_hat, A, _ = model(bag_features) 
and A is the matrix of attention scores (n_classes x n_patches).

**Comment by jeroenmollink** [comment]
Lovely, that works. Thanks!
